client/src/index.js

```diff
@@ -31,7 +31,7 @@ const documentDataElement = document.getElementById("documentdata");
 if (documentDataElement) {
```

Comment:
Would you mind replacing `documentData` with `docData` which I think reads better (even though it's cryptic sounding word) because it'd feel more consistent and further insist on staying awat from the word "document" from "window.document".